kebab_city_tour/src/LoginPage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './index.css';
+import Header from './Header';
+
+export default function LoginPage() {
+  return (
+    <div className="min-h-screen bg-gray-100">
+      <Header />
+      <main className="p-8">
+        <div className="bg-gray-300 w-full h-96 flex items-center justify-center">
+          <h1 className="text-5xl font-bold">Login</h1>
+        </div>
+      </main>
+    </div>
+  );
+}
```

Comment:
This is dumy page, but really so wide login will be ? i can't image this

Reply:
I think Kacper has only done the sketch for now, but you're right. We won't put it on main bcs it's not finished

Reply:
Why won't push on main??
Give a specific reason. I see it like it is just a skech to check if routing is done well, header displays correctly and to prepare a Login site. Especially if only I am working on this so it will be changed as well later

Reply:
If u change things we gave in the reviews and you still want to put it on main, you have to write a new issue to finish the main page etc. Later you will no longer be able to add a new issues. Also if our PM agrees to add it to main, it's ok for me.

Reply:
Also there is issue #11 to create a Login form

Reply:
Well if PR is named "main page" then why do login page anyway ? You could just focus on creation the best Main page you could do, and link it later. I don't get it why we should accept this and push it to main just beacouse "Ohh you know this will be changed later, so accept me this"

Reply:
Because boss, my main intention was to check the routing to LoginPage and in the process I threw myself a blank square.

Reply:
✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️